docs/unloading/Export.md

```diff
@@ -1,6 +1,6 @@
 # Export data using EXPORT
 
-This topic describes how to export data from specified tables or partitions in your StarRocks cluster as CSV data files by using the [EXPORT](../sql-reference/sql-statements/data-manipulation/EXPORT.md) statement to HDFS, Alibaba Cloud OSS, AWS S3, or other S3-compatible object storage systems.
+This topic describes how to export data from specified tables or partitions in your StarRocks cluster as CSV data files by using the [EXPORT](../sql-reference/sql-statements/data-manipulation/EXPORT.md) statement to an external storage system, which can be a distributed file system HDFS or a cloud storage system such as AWS S3.
```

Comment:
this sentence is a little long and complex, suggest: This topic describes how to use the [EXPORT](../sql-reference/sql-statements/data-manipulation/EXPORT.md) statement to export data from specified tables or partitions in your StarRocks cluster to an external storage system, which can be a distributed file system HDFS or a cloud storage system such as AWS S3. Data can only be exported as CSV data files.